### PR TITLE
Setup for main function integration test

### PR DESCRIPTION
**Pull request by Tulsishah**
### Description
Created SetBucketOrMntDir function.

### Testing details
1. Manual - all the integration tests are working fine( for test bucket and mntDir)
2. Unit tests - N/A
3. Integration tests - N/A

### CODE REVIEW

File: tools/integration_tests/setup/setup.go [hunk 1]
```diff
@@ -208,6 +208,33 @@ func ExecuteTestForFlags(flags [][]string, m *testing.M) (successCode int) {
...
+
+	if *testBucket == "" && *mountedDirectory == "" {
+		log.Printf("--testbucket or --mountedDirectory must be specified")
+		os.Exit(0)
+	} else if *testBucket != "" && *mountedDirectory != "" {
+		log.Printf("Both --testbucket and --mountedDirectory can't be specified at the same time.")
+		os.Exit(0)
+	}
+
+	if *mountedDirectory != "" {
```
Comment (vadlakondaswetha) [inline tools/integration_tests/setup/setup.go:222]: please add comments to the code.

File: tools/integration_tests/setup/setup.go [hunk 2]
```diff
@@ -208,6 +208,33 @@ func ExecuteTestForFlags(flags [][]string, m *testing.M) (successCode int) {
...
+		mntDir = *mountedDirectory
+		successCode := ExecuteTest(m)
+		os.Exit(successCode)
+	}
+
+	if err := SetUpTestDir(); err != nil {
```
Comment (vadlakondaswetha) [inline tools/integration_tests/setup/setup.go:228]: here also please add commetns

**Review by vadlakondaswetha — APPROVED** [review]

File: tools/integration_tests/setup/setup.go (hunk 1 above)
Comment (Tulsishah) [reply tools/integration_tests/setup/setup.go:223]: added

File: tools/integration_tests/setup/setup.go (hunk 2 above)
Comment (Tulsishah) [reply tools/integration_tests/setup/setup.go:230]: added

**Pull request merged**